app/styles/landing.css

```diff
@@ -0,0 +1,60 @@
+.landing {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  height: 80vh;
+  width: 70%;
+  margin: auto;
+}
+
+.landing__mainHeading {
+  font-size: 2rem;
+  color: black;
+}
+
+.landing__subHeading {
+  font-size: 1.5rem;
+  color: black;
+}
+
+.landing > * {
+  padding: 12px;
+}
+
+.landing__features-lists {
+  list-style: none;
+}
+
+.landing__features-list {
+  padding: 4px;
+  font-size: 1.2rem;
+  text-align: left;
+  width: 50%;
+  margin: auto;
+}
+
+.get-started__btn a {
+  padding: 12px 32px;
```

Comment:
NIT: padding in rem will also leads to more responsive UI.

Reply:
DONE